### DISCUSSION TITLE
11th open maintainers call

### DISCUSSION THREAD

**Discussion by mxinden** [libp2p/rust-libp2p#3301] [General]
Hi everyone,

The rust-libp2p maintainers (@elenaf9, @jxs and I) will meet **tomorrow (Tuesday)**. See [libp2p calendar](https://calendar.google.com/calendar/embed?src=libp2p.io_0q9682i3te7eanhe9q7ae1c58g%40group.calendar.google.com).

https://meet.jit.si/rust-libp2p-open-maintainers-call

Everyone is welcome to join, listen along, propose topics and join the discussions.

See you all tomorrow.

**Comment by nazar-pc** [comment]
I'll try to join and would like to get feedback on https://github.com/libp2p/rust-libp2p/pull/3287 if you have time to check it before then.

**Comment by mxinden** [comment]
Couple of notes from the meeting:

- Any wishes for rust-libp2p in 2023?
- feat(kad): Limit number of active outbound streams https://github.com/libp2p/rust-libp2p/pull/3287
- Progress on rust-libp2p punchr (~50% success rate)
    - rust-libp2p
        - flush instead of close, or even none of either https://github.com/libp2p/rust-libp2p/issues/3298
        - fix(quic): Identify /quic as QUIC address #3288
    - punchr
        - use keep_alive::Behaviour to await identify.
        - prioritize QUIC connections to bootstrap nodes to learn QUIC external addresses.

**Reply by geotro** [reply]
Wishing for QUIC to soon become a first class citizen and reaching ~90% hole punching success rates :)
